Post/models.py

```diff
@@ -20,3 +21,6 @@ def __str__(self):
     @classmethod
     def all_posts(cls):
         return cls.objects.all()
+
+    def get_absolute_url(self):
```

Comment:
Why is it part of model?

Reply:
@Yarboa This function redirects to 'postList' after a post is created.
It will redirect every function that does not have a redirect configuration instead of getting an error.

Reply:
@Yarboa ?

Reply:
Sorry, still do not understand why it is part of the model? 
Do you save the location url? in the model? i do not see it either

Reply:
@Yarboa, 
This method is responsible to redirect when redirect is not defined in a view function.
Thus, this function provides a backup when a function does not have a redirect configuration (instead of getting an error).

In addition, in PostCreateView the redirect is not defined and uses this function.

Reply:
Still do not understand why it is in Model, is the default saved in Model? i do not see where, can you point me ?

Reply:
When I created 'PostCreateView', and didn't implement this function, I got an error that says:
'No URL to redirect to. Either provide a URL or define a get_absolute_url method on the Model',
I think this function suppose to be in a class that a view function has an instance of it (and CreateNewPost has an instance of a post).
Sometimes, people implement this function and use the instance ID. 

If you think it will be better to define the 'redirect' in 'CreateNewPost' function- I will do it!

Reply:
> If you think it will be better to define the 'redirect' in 'CreateNewPost' function- I will do it!

Do you think the place is in the model? It does not make any sense to me @guy9050 ?

Reply:
@Shoval-Ovadia i thought you moved it :)

Reply:
@Yarboa 
I didn't know if to remove it or no to. I will remove it now :)

Reply:
Yes please, unless you have a reason to maintain it in model, 
I do not see any reason, why it will not be part of view